src/MaterialDesignThemes.Wpf/Converters/DoubleToCornerRadiusConverter.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace MaterialDesignThemes.Wpf.Converters;
 
-internal class DoubleToCornerRadiusConverter : IValueConverter
+public class DoubleToCornerRadiusConverter : IValueConverter
```

Comment:
I propose moving this inside of a new `MaterialDesignThemes.Wpf.Converters.Internal` namespace.